src/constants/routing.ts

```diff
@@ -149,6 +149,7 @@ export const COMMON_BASES: ChainCurrencyList = {
     WRAPPED_NATIVE_CURRENCY[SupportedChainId.POLYGON_MUMBAI],
     WETH_POLYGON_MUMBAI,
   ],
+  [SupportedChainId.OPTIMISTIC_KOVAN]: [nativeOnChain(SupportedChainId.OPTIMISTIC_KOVAN)],
```

Comment:
Where did this come from? Could it have been introduced to `widgets` through an earlier merge of `main`, then deleted from `main`? I think it should be removed.

Reply:
Oh hmm, I thought it was already in. I think you're right about the source.